src/graph.cc

```diff
@@ -174,11 +174,22 @@ bool DependencyScan::RecomputeOutputDirty(Edge* edge,
     }
 
     if (output_mtime < most_recent_input->mtime()) {
+      time_t output_t = static_cast<time_t>(output_mtime);
+      time_t most_recent_input_t = static_cast<time_t>(most_recent_input->mtime());
```

Comment:
On Windows, this value is not a time_t.  I think we need a function that converts these mtimes to human-readable times that is implemented differently on Linux and Windows.